c10/core/DispatchKeySet.cpp

```diff
@@ -40,7 +40,7 @@ DispatchKeySet getRuntimeDispatchKeySet(DispatchKey t) {
     case DispatchKey::Autograd:
       return autograd_dispatch_keyset;
     case DispatchKey::CompositeImplicitAutograd:
-      return math_dispatch_keyset;
+      return math_dispatch_keyset | DispatchKeySet({DispatchKey::FuncTorchBatched});
```

Comment:
cc @ezyang -- is this fine or is the right thing to do here to put FuncTorchBatched into math_dispatch_keyset? From what I can tell from the code comments CompoisiteImplicitAutograd is supposed to map 1:1 with math_dispatch_keyset

Reply:
ok sure, I'll do that :)

I originally put it there but I actually moved it since I wasn't sure this should be considered a "math key"...